java-checks/src/main/java/org/sonar/java/checks/ArrayHashCodeAndToStringCheck.java

```diff
@@ -41,6 +46,27 @@ protected MethodMatchers getMethodInvocationMatchers() {
   @Override
   protected void onMethodInvocationFound(MethodInvocationTree mit) {
     String methodName = mit.methodSymbol().name();
-    reportIssue(ExpressionUtils.methodName(mit), "Use \"Arrays." + methodName + "(array)\" instead.");
+    QuickFixHelper.newIssue(context)
+      .forRule(this)
+      .onTree(ExpressionUtils.methodName(mit))
+      .withMessage("Use \"Arrays." + methodName + "(array)\" instead.")
+      .withQuickFixes(() -> getQuickFix(mit))
```

Comment:
The method name has already been computed, you might want to re-use it later in your quickfixes builder